soco/events_asyncio.py

```diff
@@ -452,6 +452,7 @@ async def unsubscribe(
                 return
             await unsub
         except Exception as exc:  # pylint: disable=broad-except
+            self._cancel_subscription(exc)
```

Comment:
This mirrors the exception handling in `subscribe`.

Throwing away the subscription makes sense here since its clearly not usable
if we get an exception.